client/__tests__/e2e/e2e.test.ts

```diff
@@ -545,8 +563,8 @@ for (const testDataset of testDatasets) {
 
           expect(initialCount).toBe(panzoomLasso.count);
 
-          await page.getByTestId("mode-pan-zoom");
-          await page.getByTestId("mode-lasso");
+          page.getByTestId("mode-pan-zoom");
+          page.getByTestId("mode-lasso");
```

Comment:
Not sure either, this was just a result of me removing unnecessary awaits

Reply:
I think these are either preparation for something else or if the intention is to just verify the visibility then it should be updated to:

```
await expect(page.getByTestId("mode-pan-zoom")).toBeVisible();
await expect(page.getByTestId("mode-lasso")).toBeVisible();
```

Otherwise these are no-op and can be safely removed in my opinion

Reply:
took a look at the test and looks like its been broken for some time. Fixed it